plugins/modules/ec2_vol.py

```diff
@@ -439,6 +451,26 @@ def update_volume(module, ec2_conn, volume):
             volume["multi_attach_enabled"] = response.get("VolumeModification").get("TargetMultiAttachEnabled")
             volume["throughput"] = response.get("VolumeModification").get("TargetThroughput")
 
+            if module.params.get("wait"):
+                mod_state = ""
+                _wait_till = module.params.get("wait_timeout") + time.time()
+                while _wait_till > time.time():
+                    mod_response = ec2_conn.describe_volumes_modifications(VolumeIds=[volume["volume_id"]])
```

Comment:
One small thing. I guess we should enclose  mod_response = ec2_conn.describe_volumes_modifications(VolumeIds=[volume["volume_id"]]) in a try-except block.

Reply:
Ok, I can add that